test/ibmi.connection.test.js

```diff
@@ -5,70 +5,61 @@
 
 'use strict';
 
-var describe = require('./describe');
+const describe = require('./describe');
 
 /* eslint-env node, mocha */
 process.env.NODE_ENV = 'test';
 require('./init.js');
-var assert = require('assert');
-var DataSource = require('loopback-datasource-juggler').DataSource;
+const assert = require('assert');
+const DataSource = require('loopback-datasource-juggler').DataSource;
 
-var config;
+let config;
 
 before(function() {
   config = global.config;
 });
 
 describe('testConnection', function() {
   it('should pass with valid settings', function(done) {
-    var db = new DataSource(require('../'), config);
+    const db = new DataSource(require('..'), config);
     db.ping(function(err) {
       assert(!err, 'Should connect without err.');
-      done();
+      db.disconnect(done);
```

Comment:
If it's important to disconnect after each test, then please move this code to an `afterEach` hook. That way you don't have to repeat yourself (and perhaps accidentally forget to disconnect in one of the many tests).

What's more important, this line is not executed when any of the assertions above failed! I.e. when a test fails, the datasource is not disconnected, which may cause cascading failures in follow-up tests that would otherwise pass.

The hook `beforeEach` is executed always, regardless of the test outcome (pass/fail).

```js
beforeEach(() => db.disconnect());
```

(Promise mode is supported both by `db.disconnect` and Mocha.)